tests/regression/01-cpa/28-interval.c

```diff
@@ -1,4 +1,4 @@
-// PARAM: --set solver "'new'" --set ana.int.interval true
```

Comment:
Why remove this? i think it is one of the very few (if only) tests of the `new` solver

Reply:
Look at this test, it's for testing fundamental interval analysis that should in no way be specific to some historical solver. If you want to test non-default solvers, then you can change the default and run the tests. The removal of these hardcoded `PARAM`s makes it possible to actually run everything with alternative defaults. As it is hardcoded, there's no way to run this with TD3. "New" is far from the only solver that we're not automatically testing right now. To test all solvers properly, you'd have to do a testing matrix with all the different solvers as default and rerun the whole suite, but that hasn't been nor is the goal right now.

Reply:
Yes, but I'd still be opposed to leaving things worse than they are currently :wink:

Reply:
I'd prefer we test the solver we actually use, not some random old solver though.

Reply:
I added a copy of this test with alternative `PARAM`.